examples/examples/tower-load.rs

```diff
@@ -1,408 +1,412 @@
-//! A demo showing how filtering on values and dynamic filter reloading can be
```

Comment:
This one is failing, but we should get it building again on `std::future::Future`. I think `tokio-compat` can help a lot here.